dawn/src/dawn/AST/ASTExpr.h

```diff
@@ -500,6 +500,14 @@ class FieldAccessExpr : public Expr {
   //  - argumentMap_      : {1, -1, -1}       // `dir` maps to the 1st argument of `avg` (0 based)
   //  - argumentOffset_   : {1, 0, 0}         // `dir+1` has an offset `+1`
```

Comment:
can you somehow put in here that `-1 ` means not set for map and 0 means not set for offset?

Reply:
done